configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -860,9 +861,55 @@
 
             </div>
 
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.judgement"/>
+                    </label>
+                    <obs conceptId="CIEL:167116" style="radio" answerConceptIds="PIH:1897,CIEL:159406,CIEL:159407" answerCodes="good,fair,poor" answerSeparator=""/>
+                </p>
+
+            </div>
+
         </div>
+        <!-- No Concept Yet-->
+        <div class="section-container" id="radio_button_container">
 
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.danger"/>
+                    </label>
+                    <obs conceptId="PIH:15106" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
```

Comment:
answerCodes is not using the message properties.  This code will show the answers in English (yes/no).

Remove answerCodes.  It will show the concept preferred names:  yes,no (English) and oui,non

Reply:
@lnball I made the change